signup-service/src/test/java/io/mosip/signup/services/RegistrationServiceTest.java

```diff
@@ -533,7 +533,7 @@ public void doVerifyChallenge_withInvalidFormatForOTPChallenge_thenFail() {
     }
 
     @Test
-    public void doVerifyChallenge_toResetPassword_thenIdentityInactive() {
+    public void doVerifyChallengeInResetPassword_thenIdentityInactive_throwException() {
```

Comment:
it is better if we can rename this to "doVerifyChallengeInResetPassword_withInactiveIdentity_throwException()"